Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.18.1-alpine3.16@sha256:868e2b6c8923d87a4bbfb157d757898061c9000aaaedf64472074fa7b62d0e72
+FROM node:16.19.1-alpine3.17@sha256:25828d5c4ae9824273db9ca2e923da2d29bbae78f534e979f09eb99a2e812e94
```

Comment:
That's weird, the SHA when I download this starts with `0fcf4f`

Reply:
It's `25828d5c4ae9824273db9ca2e923da2d29bbae78f534e979f09eb99a2e812e94` at https://hub.docker.com/layers/library/node/16.19.1-alpine3.17/images/sha256-25828d5c4ae9824273db9ca2e923da2d29bbae78f534e979f09eb99a2e812e94?context=explore